### ISSUE TITLE
[Bug] logout, hospitals and profile option not showing on navbar after login 

### ISSUE THREAD

**Issue by Luson045** [Luson045/medi-connect#366]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug Description?

<h1>BEFORE     ||    NOW</h1>
<div style="display:flex">
<img width="227" alt="image" src="https://github.com/user-attachments/assets/8fc517b3-4816-4d30-a548-03614465409c">
<img width="239" alt="image" src="https://github.com/user-attachments/assets/3b9c605d-68d0-4ac5-b2f3-32ce10cbfc31">
</div>

### Record

- [X] I agree to follow this project's Code of Conduct
- [ ] I'm a GSSOC contributor
- [ ] I want to work on this issue (Optional)
- [ ] I'm willing to provide further clarification or assistance if needed.

**Comment by Crimson-Typhoon-147** [comment]
Assign it to me . I can solve this.

**Comment by harish08102004** [comment]
Hey @Luson045 I can work on this.

**Comment by shubhagarwal1** [comment]
Hey @Luson045 i think there is a problem with navbar component in the application, but one thing are you talking about diffrent hospital option as here is hospitals around but yes no profile, i think i can fix this can you please assign me this task
<img width="348" alt="Screenshot 2024-10-11 at 5 02 17 AM" src="https://github.com/user-attachments/assets/7b0dbeb8-41d3-4cc7-9fa2-dde9082aaf7f">

**Comment by Luson045** [comment]
@Crimson-Typhoon-147  okay 👍🏻

**Comment by Luson045** [comment]
Yeah @shubhagarwal1 there's one more page called hospitals you can see it by url https://med-space.vercel.app/hospitals 
It's not there in the navbar anymore, someone messed it.

**Comment by lade6501** [comment]
Hey @Crimson-Typhoon-147 just for info Navbar component is taking 3 props `isAuthenticated`, `user` and `handleLogout` you just need to pass these value to `Navbar`component

**Comment by shubhagarwal1** [comment]
> Yeah @shubhagarwal1 there's one more page called hospitals you can see it by url https://med-space.vercel.app/hospitals 
> It's not there in the navbar anymore, someone messed it.

Okay i understand can you please assign me this task

**Comment by Luson045** [comment]
@Crimson-Typhoon-147 updates

**Comment by Crimson-Typhoon-147** [comment]
Working on it @Luson045 . It'll be completed by tonight.

**Comment by Luson045** [comment]
great 👍
